consensus_decentralization/helper.py

```diff
@@ -265,24 +265,24 @@ def get_metrics_config():
     return metrics
 
 
-def get_default_ledgers():
+def get_ledgers():
     """
-    Retrieves data regarding the default ledgers to use
+    Retrieves data regarding the ledgers to use
     :returns: a list of strings that correspond to the ledgers that will be used (unless overriden by the relevant cmd
     arg)
     """
     config = get_config_data()
-    ledgers = config['default_ledgers']
+    ledgers = config['ledgers']
     return ledgers
 
 
-def get_default_start_end_dates():
+def get_start_end_dates():
     """
     Retrieves the start and end dates for which to analyze data
     :returns: a tuple of two strings, (<start date>, <end date>)
     """
     config = get_config_data()
-    return str(config['default_timeframe']['start_date']), str(config['default_timeframe']['end_date'])
+    return str(config['timeframe']['start_date']), str(config['timeframe']['end_date'])

```

Comment:
With the cmd args we used to make sure that the given strings were valid dates. We could also do this here by calling `valid_date()`

Reply:
fixed